server/src/bots/telegram/index.ts

```diff
@@ -1,53 +1,55 @@
+import { showAdvicesHowToBehaveResponse } from './botResponse/adviceResponse';
+import { showHelpInfoResponse } from './botResponse/helpResponse';
+import { Express } from 'express';
+import { cachedCovid19CountriesData, getAvailableCountries } from '../../services/domain/covid19';
+import { Country } from '../../models/country.models';
+import { flag } from 'country-emoji';
+import { assistantStrategyResponse } from './botResponse/assistantResponse';
 import * as TelegramBot from 'node-telegram-bot-api';
 import Config from '../../environments/environment';
-import {logger} from '../../utils/logger';
-import {startResponse} from './botResponse/startResponse';
-import {showAvailableCountriesResponse} from './botResponse/availableResponse';
+import { logger } from '../../utils/logger';
+import { startResponse } from './botResponse/startResponse';
+import { showAvailableCountriesResponse } from './botResponse/availableResponse';
 import {
     showExistingSubscriptionsResponse,
     subscribingStrategyResponse,
-    subscriptionManagerResponse
+    subscriptionManagerResponse,
 } from './botResponse/subscribeResponse';
-import {SubscriptionType} from '../../models/subscription.models';
-import {registry, withCommandArgument} from './services/messageRegistry';
-import {subscriptionNotifierHandler} from './services/subscriptionNotifierManager';
-import {unsubscribeStrategyResponse} from './botResponse/unsubscribeResponse';
-import {showTrendsByCountry} from './botResponse/trendResponse';
-import {CountrySituationInfo} from '../../models/covid19.models';
-import {catchAsyncError} from '../../utils/catchError';
-import {LogglyTypes} from '../../models/loggly.models';
-import {getErrorMessage} from '../../utils/getLoggerMessages';
+import { SubscriptionType } from '../../models/subscription.models';
+import { MessageHandlerRegistry, withCommandArgument } from './services/messageHandlerRegistry';
+import { subscriptionNotifierHandler } from './services/subscriptionNotifierManager';
+import { unsubscribeStrategyResponse } from './botResponse/unsubscribeResponse';
+import { showTrendsByCountry } from './botResponse/trendResponse';
+import { CountrySituationInfo } from '../../models/covid19.models';
+import { catchAsyncError } from '../../utils/catchError';
+import { LogglyTypes } from '../../models/loggly.models';
+import { getErrorMessage } from '../../utils/getErrorMessages';
 
-function runTelegramBot(app: Express, ngRokUrl: string) {
+function runTelegramBot(app: Express, appUrl: string) {
     // Create a bot that uses 'polling' to fetch new updates
-    const bot = new TelegramBot(Config.TELEGRAM_TOKEN, {polling: true});
+    const bot = new TelegramBot(Config.TELEGRAM_TOKEN, { polling: true });
+    const messageHandlerRegistry = new MessageHandlerRegistry(bot);
 
     // This informs the Telegram servers of the new webhook
-    bot.setWebHook(`${ngRokUrl}/bot${Config.TELEGRAM_TOKEN}`);
+    bot.setWebHook(`${appUrl}/bot${Config.TELEGRAM_TOKEN}`);
```

Comment:
@danbilokha do you mind adding ```TELEGRAM_TOKEN``` as a parameter.
That we we will have this index independent from environment but only from parameters passed

Reply:
done